.travis.yml

```diff
@@ -8,11 +8,6 @@ cache:
     - $HOME/.pub-cache
 jobs:
   include:
-    - dart: 2.4.1
-      name: "SDK: 2.4.1"
-      script:
-        - pub run dart_dev analyze
-        - pub run dart_dev test --release
```

Comment:
Could we just bump this to what we're currently using internally (2.7.2) instead of removing it altogether? I've tried to keep this option in place for our OSS repos since we won't always upgrade right away and OSS consumers might not, either.

Reply:
Or, alternatively, if something is prohibiting us from running on these older versions, we should update our SDK constraint in `pubspec.yaml`

Reply:
Yep! I'll update that!